src/components/pages/Fuse/FuseReferralBar.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button } from "@chakra-ui/react";
+import { RowOrColumn, Center } from "buttered-chakra";
+import React from "react";
+import { useParams } from "react-router-dom";
+import { useIsSmallScreen } from "../../../hooks/useIsSmallScreen";
+import DashboardBox from "../../shared/DashboardBox";
+import { mediumAddress } from "../../../utils/shortAddress";
+import { useRari } from "../../../context/RariContext";
+
+const FuseReferralBar = () => {
+    const isMobile = useIsSmallScreen();
+    let { poolId } = useParams();
+    const { address } = useRari();
+
+    const short_referral_code = `https://app.rari.capital/fuse/pool/${poolId}/${mediumAddress(address)}`;
+    const full_referral_code = `https://app.rari.capital/fuse/pool/${poolId}/${address}`;
+
+    let width = !isMobile ? "50%" : "100%";
+    let left_padding = !isMobile ? "0.5rem" : "";
+    
+    return (
+        <RowOrColumn
+            isRow={!isMobile}
+            expand
+            mainAxisAlignment="flex-end"
+            crossAxisAlignment="center"
+            py={4}
+            >
+            <div style={{ width: width, paddingLeft: left_padding}}>
```

Comment:
Can we avoid using a div here and just add those width and padding properties to the child `DashboardBox`?

Reply:
I think I need this because padding the box makes the width larger and I can't control that with the margin because it expands as margin is not counted in the 50% width

Reply:
ohhh yeah i thought i did that somewhere as well.